scripts/functions/indexes/elasticsearch.py

```diff
@@ -38,6 +52,27 @@ def check_status_eleasticsearch(url):
         print(e)
         return False
 
+def request_elasticsearch(method, url):
+    try:
+        print(method + " " + url)
+        result = requests.request(method, url)
+        if not result.ok :
+            # TODO this might not be a json format
```

Comment:
isn't it handled with the prettyprint()?

Reply:
true, comment moved in `prettyprint` Function